src/ParallaxSwiper.js

```diff
@@ -94,7 +105,7 @@ class ParallaxSwiper extends Component {
             width: vertical
               ? this.state.width
               : this.state.width + dividerWidth,
-            height: this.state.height,
+            height: heigth?heigth:this.state.height,
```

Comment:
Not your fault, but I need to add a .prettierrc file so formatting will be correct.